axum/src/extract/multipart.rs

```diff
@@ -142,6 +142,38 @@ impl<'a> Field<'a> {
     pub async fn text(self) -> Result<String, MultipartError> {
         self.inner.text().await.map_err(MultipartError::from_multer)
     }
+
+    /// Stream a chunk of the field data.
+    ///
+    /// When the field data has been exhausted, this will return [`None`].
+    ///
+    /// This does the exact same thing as the `Stream` impl and is for convenience
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use axum::{
+    ///    extract::Multipart,
+    ///    routing::post,
+    ///    Router,
+    /// };
+    ///
+    /// async fn upload(mut multipart: Multipart) {
+    ///     while let Some(mut field) = multipart.next_field().await.unwrap() {
+    ///         while let Some(chunk) = field.chunk().await.unwrap() {
```

Comment:
We should handle the errors and not use `unwrap`. Users tend to copy/paste docs verbatim so need to make sure we lead by example.

I think `.map_err(|err| (StatusCode::BAD_REQUEST, err.to_string())?` is fine.

Reply:
fixed